zip-0226.rst

```diff
@@ -298,6 +314,27 @@ Burn Mechanism Consensus Rules
 
 **Note:** Even if this mechanism allows having transparent ↔ shielded Asset transfers in theory, the transparent protocol will not be changed with this ZIP to adapt to a multiple Asset structure. This means that unless future consensus rules changes do allow it, unshielding will not be possible for Custom Assets.
 
+Rationale for Burn Mechanism
+````````````````````````````
+
+As for the value balance verification, we assume :math:`n` Actions, :math:`l` of which are related to ZEC and the remaining :math:`n-l` Actions related to Custom Assets.
+
+The burn mechanism verification is equivalent to:
+
+.. math:: \mathsf{bvk} = (\mathsf{(\sum_{i=1}^{l} cv_i^{net})} + \mathsf{(\sum_{j=l+1}^{n} cv_i^{net})}) - ([\mathsf{v^{balanceOrchard}}]\mathcal{V}^{\mathsf{Orchard}} + [0]\mathcal{R}^{\mathsf{Orchard}}) - (\sum_{k=1}^{|\mathsf{assetBurn}|} [\mathsf{v_{k}^{AssetBase}}]\mathsf{AssetBase_{k}} + [0]\mathcal{R}^{\mathsf{Orchard}})
+
+Removing the points at infinity (i.e. identity elements in the group) and unrolling the :math:`cv_i`'s:
+
+.. math:: \mathsf{bvk} = ((\sum_{i=1}^{l} [\mathsf{v_i^{net}}]\mathcal{V}^{\mathsf{Orchard}} + [\mathsf{rcv_i}]\mathcal{R}^{\mathsf{Orchard}}) + (\sum_{j=l+1}^{n} \mathsf{[v_j^{net}]\mathsf{AssetBase}_j} + [\mathsf{rcv_j}]\mathcal{R}^{\mathsf{Orchard}})) - ([\mathsf{v^{balanceOrchard}}]\mathcal{V}^{\mathsf{Orchard}}) - (\sum_{k=1}^{|\mathsf{assetBurn}|} [\mathsf{v_{k}^{AssetBase}}]\mathsf{AssetBase_{k}})
+
+Rearranging
+
+.. math:: \mathsf{bvk} = [(\sum_{i=1}^{l} [\mathsf{v_i^{net}}]\mathcal{V}^{\mathsf{Orchard}} + [\mathsf{rcv_i}]\mathcal{R}^{\mathsf{Orchard}}) - ([\mathsf{v^{balanceOrchard}}]\mathcal{V}^{\mathsf{Orchard}})] + [(\sum_{j=l+1}^{n} \mathsf{[v_j^{net}]\mathsf{AssetBase}_j} + [\mathsf{rcv_j}]\mathcal{R}^{\mathsf{Orchard}}) - (\sum_{k=1}^{|\mathsf{assetBurn}|} [\mathsf{v_{k}^{AssetBase}}]\mathsf{AssetBase_{k}})]
```

Comment:
The unrolling is unnecessary for a reader that understand the original nu5 spec. It tolls the reader and will require a significant amount of maintenance from us in the future. Remove those lines.